packages/fast-components-class-name-contracts-base/src/stack-panel/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * The class name contract for the stackpanel component
+ */
+export interface StackPanelClassNameContract {
+    /**
+     * The root of the stackpanel component
+     */
+    stackPanel?: string;
+
+    /**
+     * the container that contains the stack panel items
+     */
+    stackPanel_itemContainer?: string;
+
+    /**
+     *
+     */
+    stackPanel__isScrolling?: string;
```

Comment:
just as we would say `stackPanel__active`, and not `stackPanel__isActive` I prefer `stackPanel__scrolling`.

Also, think you need a comment above this.

Reply:
done